pages/linux/snap.md

```diff
@@ -15,6 +15,10 @@
 
 `snap refresh {{package_name}}`
 
+- Update a package to another channel (track, risk-level, or branch):
```

Comment:
What is `track, risk-level, or branch`?
```suggestion
- Update a package to another channel:
```

Reply:
I wasn't sure if the term channel was going to widely understood enough, so I thought that might help make it more explicit.

Track: A type of release, by default it's `latest`, I've never seen this used in any way tbh.
Risk-Level: Stable / Beta / Edge
Branch: For debug releases, like a one-off version of a particular track/risk-level to fix a particular issue or that allows a user to test a specific new feature without giving it to everyone.

I'd like to wait for other opinions, if they agree to remove it, then that's cool.

Reply:
Maybe
```suggestion
- Update a package to another channel (e.g. stable, beta or development):
```
Not sure if we should explain the concept of `channel`.

Reply:
What are the possible valid values?

Reply:
It's not fixed, the publisher can use whatever their wants. Node uses `14/stable`, intellij `edge`, `beta`, `stable`.

Reply:
Ah, I was afraid so.  No suggestions on my part, then :(

Reply:
Generally here we want to use the official terminology that the application itself uses

Reply:
I assume that means we're deciding between leaving it at channel, or specifying track, risk-level, or branch?

Opinion on each?
1️⃣ Update a package to another channel:
2️⃣ Update a package to another channel (track, risk-level, or branch):
3️⃣ Seth, you misunderstood. Your ideas are bad, and you should feel bad.

Reply:
Ahh so sad, I was hoping people could react with 1️⃣, 2️⃣, and 3️⃣. Seems we're limited to just the 8 GitHub give us.

Reply:
Yeah, GitLab is much better in the regard 😢

Reply:
What's the official terminology used by the snapcraft website?

Reply:
![image](https://user-images.githubusercontent.com/22801583/119654604-eed77900-be28-11eb-8d4e-42f368278a29.png)

In the docs I linked in the PR description

Reply:
e.g. `14/stable` for Node.js and `latest/beta`, `2021.1/stable` for IntelliJ. Cool. To make it explicit: The combination of `<track>/<risk>/<branch>` is the name of the channel.

Reply:
In that case, we should probably use that terminology then, to avoid confusing things.

Reply:
So, we're happy with something like the following?

```suggestion
- Update a package to another channel (track, risk, or branch):
```

Reply:
Leaving unresolved on purpose, as this looks like an important conversation.